CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## Release Notes
 
+### 0.24.3
```

Comment:
Since this is a new feature, this should be 0.25.0 according to the semantic revision definition